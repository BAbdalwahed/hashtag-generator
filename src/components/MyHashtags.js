import React from "react";

const MyHashtags = () => {
  return (
    <div className="my-hashtags">
      <h1>#My Hashtags</h1>
      <form className="my-hashtags__form">
        <label htmlFor="category-filter">Filter Hashtags by Category{}</label>
        <select id="category-filter">
          <option value="Music">Music</option>
          <option value="Art">Art</option>
          <option value="Travel">Travel</option>
          <option value="Art">Art</option>
          <option value="Business">Business</option>
          <option value="Other">Other</option>
        </select>
      </form>
      <div className="my-hashtags__display">
        <div className="my-hashtags__card">
          <h2>Card Title</h2>
          <p>
            #nowplaying #love #radio #video #live #hiphop #spotify #newmusic
            #life #apple #rock #dance #christmas #playlist #pop #musicvideo
            #song #applemusic #goodmusic #time #country #playing #album
            #listening #musical #songs #christmasmusic #likedvideo
            #officialmusic #officialmusicvideo
          </p>
          <button type="button">Copy</button>
        </div>
      </div>
    </div>
  );
};

export default MyHashtags;
