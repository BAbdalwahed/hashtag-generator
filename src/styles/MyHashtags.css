.Myhashtags {
    justify-content: center;
}
.addNewHashtag {
    display: flex;
    justify-content: center;
    margin: auto;
}