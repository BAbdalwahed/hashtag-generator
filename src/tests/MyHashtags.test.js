import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from "react-router-dom";
import MyHashtags from "../components/MyHashtags";

xit("renders Login component", () => {
  const { asFragment } = render(
    <BrowserRouter>
      <MyHashtags />
    </BrowserRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
