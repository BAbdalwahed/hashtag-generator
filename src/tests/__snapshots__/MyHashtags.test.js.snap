// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Login component 1`] = `
<DocumentFragment>
  <div
    class="my-hashtags"
  >
    <h1>
      #My Hashtags
    </h1>
    <form
      class="my-hashtags__form"
    >
      <label
        for="category-filter"
      >
        Filter Hashtags by Category
      </label>
      <select
        id="category-filter"
      >
        <option
          value="Music"
        >
          Music
        </option>
        <option
          value="Art"
        >
          Art
        </option>
        <option
          value="Travel"
        >
          Travel
        </option>
        <option
          value="Art"
        >
          Art
        </option>
        <option
          value="Business"
        >
          Business
        </option>
        <option
          value="Other"
        >
          Other
        </option>
      </select>
    </form>
    <div
      class="my-hashtags__display"
    >
      <div
        class="my-hashtags__card"
      >
        <h2>
          Card Title
        </h2>
        <p>
          #nowplaying #love #radio #video #live #hiphop #spotify #newmusic #life #apple #rock #dance #christmas #playlist #pop #musicvideo #song #applemusic #goodmusic #time #country #playing #album #listening #musical #songs #christmasmusic #likedvideo #officialmusic #officialmusicvideo
        </p>
        <button
          type="button"
        >
          Copy
        </button>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
